openslides/motions/static/templates/motions/slide_motion.html

```diff
@@ -69,7 +69,7 @@ <h3 ng-if="motion.polls.length > 0 && motion.polls[0].has_votes" translate>Votin
   <!-- Title -->
   <div id="title">
     <h1>{{  motion.agenda_item.getTitle() || motion.getTitle() }}</h1>
-    <h2>
+    <h2 ng-if="config('motions_slide_has_motion_id')">
```

Comment:
Value is not defined yet. I just needed a way to exclude the motion subheader in the slide view. I'm not sure if @emanuelschuetze will approve this change request